src/Persistence/ObjectManagerDecorator.php

```diff
@@ -76,6 +81,28 @@ public function initializeObject(object $obj): void
         $this->wrapped->initializeObject($obj);
     }
 
+    public function isUninitializedObject(mixed $value): bool
+    {
+        if (! method_exists($this->wrapped, 'isUninitializedObject')) {
```

Comment:
I plan to simplify all this in a follow up PR that makes a real signature out of the `@method` annotation about `isUninitializedObject` in `ObjectManager`.